mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/PreBuildTestOptimizer.java

```diff
@@ -30,12 +30,12 @@
     @Override
     public void afterProjectsRead(MavenSession session) throws MavenExecutionException {
 
-        if (isDisableSmartTesting()) {
+        configuration = Configuration.read();
```

Comment:
Yes maybe `load` might be better I don't know which could be better in this case